cmd/kured/main.go

```diff
@@ -182,16 +193,44 @@ func flagCheck(cmd *cobra.Command, args []string) {
 		log.Warnf("Deprecated flag(s). Please use --notify-url flag instead.")
 		trataURL, err := url.Parse(slackHookURL)
 		if err != nil {
-			log.Warnf("slack-hook-url is not properly formatted...no notification will be sent: %v\n", err)
+			log.Warnf("slack-hook-url is not properly formatted... no notification will be sent: %v\n", err)
 		}
 		if len(strings.Split(strings.Trim(trataURL.Path, "/services/"), "/")) != 3 {
-			log.Warnf("slack-hook-url is not properly formatted...no notification will be sent: %v\n", err)
+			log.Warnf("slack-hook-url is not properly formatted... no notification will be sent: unexpected number of / in URL\n")
```

Comment:
I'm not sure if this should be changed in this PR. Maybe open another one to change the message.

Reply:
reverted